app/views/my/solutions/_nav_and_header.html.haml

```diff
@@ -18,7 +18,7 @@
         = link_to "#{@track.title} Track", [:my, @track]
       .track-progress
         -if @user_track.independent_mode?
```

Comment:
I think we should create an issue to alias `independent_mode?` methods to `practice_mode?`.

Reply:
Yeah. I'd like to actually rename the method in the database to be honest. There's loads about this bit of the codebase that I feel I could do better now I know more about it. And also that will probably change with the new Chicago project.

I'm going to leave this for now so that I can get it merged today, and with the intention of reworking it in the DB in the coming weeks. But if this is still the case in a month, shout at me and I'll alias and rename it throughout the codebase.